scripts/ci/iptables-restore

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+	key="$1"
+	case "$key" in
+		-w?*)
+			shift
+			;;
+		*)
+			POSITIONAL+=("$key")
+			shift
+			;;
+	esac
+done
+
+set -- "${POSITIONAL[@]}"
+xtables-multi iptables-restore $*
```

Comment:
in general use "$@" as you don't control IFS

but why not using "${POSITIONAL[@]}" directly ?

Reply:
ok